backend/tests/integration-async/conftest.py

```diff
@@ -127,7 +127,7 @@ def _check(steps={}, timeout=15):
                 try:
                     result = _fun(*_args, **_kwargs)
                 except:
-                    pass
+                    break
```

Comment:
we should replace this fixture in the future :/

Reply:
I agree